src/platform/android/AndroidChipPlatform-JNI.cpp

```diff
@@ -60,6 +60,9 @@ static bool JavaBytesToUUID(JNIEnv * env, jbyteArray value, chip::Ble::ChipBleUU
 namespace {
 JavaVM * sJVM = nullptr;
 JniGlobalReference sAndroidChipPlatformExceptionCls;
+jclass callbackClass;
+jmethodID onLogMessageMethod;
+jobject sLogCallback = nullptr;
```

Comment:
Rename to  `sJavaLogCallbackObject`

Reply:
I will rename it to `sJavaLogCallbackObject`.